crates/cubecl-cpu/src/compiler/passes/trigonometries_transform.rs

```diff
@@ -0,0 +1,37 @@
+use cubecl_core::{
+    ir::{Arithmetic, Instruction, Operation, Scope},
+    prelude::*,
+};
+use cubecl_opt::{IrTransformer, TransformAction};
+
+#[derive(Debug)]
+pub(crate) struct HypotTransform;
+
+impl IrTransformer for HypotTransform {
+    fn maybe_transform(&self, scope: &mut Scope, inst: &Instruction) -> TransformAction {
+        match &inst.operation {
+            Operation::Arithmetic(Arithmetic::Hypot(op)) => {
+                let mut scope = scope.child();
+                expand_hypot(&mut scope, op.lhs, op.rhs, inst.out.unwrap());
```

Comment:
From the imports, I guess the `expand_hypot` function comes from the prelude. However I don't think we should include that in the prelude, a normal import would be better.